scrapyd_k8s/settings.py

```diff
@@ -0,0 +1 @@
+TIME_FORMAT = '%Y-%m-%d %H:%M:%S.%f'
```

Comment:
I'm not really sure if `settings` is a good name here, I would think this has to do with user-configurable settings.
Maybe `const` or `util`?
Very good to centralize this somewhere!

Reply:
I was struggling with the location too. Since there is not yet a common place to put these things. We already have a `utils.py`, but not sure if this would be a right fit there?

Reply:
I think that makes sense, let's do that.
We _could_ also make it into a function that does the `strftime`.